frame/scored-pool/src/lib.rs

```diff
@@ -264,10 +276,10 @@ pub mod pallet {
 			// Sorts the `Pool` by score in a descending order. Entities which
 			// have a score of `None` are sorted to the beginning of the vec.
 			pool.sort_by_key(|(_, maybe_score)| Reverse(maybe_score.unwrap_or_default()));
-
+			let pool_bounded: PoolT<T, I> = BoundedVec::truncate_from(pool);
```

Comment:
Not quite happy with the truncating, should be resolved after https://github.com/paritytech/substrate/pull/11998.  
Hopefully gets merged quickly

Reply:
You can approve it if you like, then we can merge it :smile: